src/Peachpie.Runtime/Reflection/PhpRoutineInfo.cs

```diff
@@ -63,6 +63,10 @@ public abstract class RoutineInfo : IPhpCallable, ICloneable
         /// <summary>Target instance when binding the MethodInfo call.</summary>
         internal virtual object Target => null;
 
+        /// <summary>Creates an <see cref="IPhpCallable"/> from an instance method, binding the target to call the method on.</summary>
+        /// <remarks>Returns <c>null</c> if not applicable.</remarks>
+        public virtual IPhpCallable BindTarget(object target) => null;
```

Comment:
remove

whenever needed, use 
```c#
PhpCallback.Create(target, callable)
```

Reply:
OK, updated in 2828173f318c841df8a368d1e4817d6ec702e2b6.